scripts/inference

```diff
@@ -18,4 +18,11 @@ then
 fi
 
 distDir=$myDir"/../dist"
-eval "java -classpath "$distDir"/checker.jar:"$distDir"/plume.jar:"$distDir"/checker-framework-inference.jar  checkers.inference.InferenceLauncher " "$@"
+classpath="$distDir"/checker.jar:"$distDir"/plume.jar:"$distDir"/checker-framework-inference.jar
+
+if [ "$thrid_party_checker_classpath" != "" ] ; then
```

Comment:
Who sets this "thrid_party_checker_classpath" environment variable? How can a developer insert their checker's classpath to this variable?